command/CommandHelper.js

```diff
@@ -0,0 +1,47 @@
+const Database = require('../data/database');
```

Comment:
I like having this in a separate file... it seems like it's a nice matching chunk of functionality... Shud we grow this file or rename it to something like AudioFileStorageHelper.js?  or some other better name?  I haven't really looked to see if there are other "helper" functions in CommandController that could be clubbed into smaller files to avoid big files.

Reply:
Yeah that's a good point. Let me look ahead to the other functions we need to pull out in the future and I'll rename this to something more specific to audio.

Reply:
I went with AudioFileHelper. There are a few more functions that fall under that umbrella which I'll move in later.